lib/js/components/Well/Well.vue

```diff
@@ -6,6 +6,19 @@
 			'-ds-small': WELL_PADDINGS.SMALL === padding,
 		}"
 	>
+		<div v-if="hasChip" class="ds-well__chipContainer">
+			<chip
+				:label="chipLabel"
+				:is-label-uppercase="chipLabelUppercase"
+				:left-icon="chipLeftIcon"
+				:radius="chipRadius"
+				:color="chipColor"
+				:color-hex="chipColorHex"
+			>
+				<slot name="chipAccessory" />
```

Comment:
I don't quite understand the slot in here? I don't think it can be used in any way? If we want to expose the accessory we would need to wrap slot in template with the correct name and then expose slot in it